cedar-policy-core/src/parser/err.rs

```diff
@@ -31,11 +31,12 @@ use thiserror::Error;
 
 use crate::ast::{self, ReservedNameError};
 use crate::parser::fmt::join_with_conjunction;
-use crate::parser::loc::Loc;
+use crate::parser::loc::{Loc, MaybeLoc};
 use crate::parser::node::Node;
 use crate::parser::unescape::UnescapeError;
 
 use super::cst;
+use super::loc::AsLocRef;
```

Comment:
inconsistent that we import this from `super::loc` but other items from `crate::parser::loc`

Reply:
Fixed - we don't use `loc` imports anymore.